Example_Notebooks/Script_DAHITI_io.ipynb

```diff
@@ -0,0 +1,992 @@
+    "import dahiti_io\n",
+    "import geopandas as gpd\n",
+    "import glob\n",
+    "import os"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "d00758c6-3a13-4e07-9559-a8c01295554f",
+   "metadata": {},
+   "source": [
+    "## Case: Lake Tana"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "ffa73d8d-fa0d-4cfe-aa20-0210f6a3fa9b",
+   "metadata": {},
+   "source": [
+    "##### The **download_DAHITI_stations** function takes two arguments. One is the folder path where data from DAHITI will be downloaded another is shapefile / geojson file of the Study Area. Geojson file of any area can be developed using this webapp: https://geojson.io/#map=9.35/24.5561/89.6891"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 2,
+   "id": "7e7ddb68-37d6-49d8-9f8d-fd7f6213eeb3",
+   "metadata": {
+    "tags": []
+   },
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "Obtain your API key from DAHITI: (https://dahiti.dgfi.tum.de/en/frequently-asked-questions/api-key/)\n"
+     ]
+    },
+    {
+     "name": "stdin",
+     "output_type": "stream",
+     "text": [
+      "Enter your API key:  D133490744EE01A16B19B0B35AE098C767B9FB009BAD9963DD73D8E5E3BD9CE9\n"
+     ]
+    },
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "Dataset(s) found: 1\n",
+      "Downloading... 110 -> b'Tana, Lake' ( /home/jovyan/Temporary Projects/XX_Thesis_Works/01_pyaltm/Lake_Tana/DAHITI_Raw/110.csv )\n",
+      "   Dahiti_ID  Longitude  Latitude\n",
+      "0        110       37.3     11.95\n"
+     ]
+    },
+    {
+     "data": {
+      "text/html": [
+       "<div>\n",
+       "<style scoped>\n",
+       "    .dataframe tbody tr th:only-of-type {\n",
+       "        vertical-align: middle;\n",
+       "    }\n",
+       "\n",
+       "    .dataframe tbody tr th {\n",
+       "        vertical-align: top;\n",
+       "    }\n",
+       "\n",
+       "    .dataframe thead th {\n",
+       "        text-align: right;\n",
+       "    }\n",
+       "</style>\n",
+       "<table border=\"1\" class=\"dataframe\">\n",
+       "  <thead>\n",
+       "    <tr style=\"text-align: right;\">\n",
+       "      <th></th>\n",
+       "      <th>Dahiti_ID</th>\n",
+       "      <th>Longitude</th>\n",
+       "      <th>Latitude</th>\n",
+       "    </tr>\n",
+       "  </thead>\n",
+       "  <tbody>\n",
+       "    <tr>\n",
+       "      <th>0</th>\n",
+       "      <td>110</td>\n",
+       "      <td>37.3</td>\n",
+       "      <td>11.95</td>\n",
+       "    </tr>\n",
+       "  </tbody>\n",
+       "</table>\n",
+       "</div>"
+      ],
+      "text/plain": [
+       "   Dahiti_ID  Longitude  Latitude\n",
+       "0        110       37.3     11.95"
+      ]
+     },
+     "execution_count": 2,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "dahiti_io.download_DAHITI_stations(\"./Lake_Tana/\",\"Lake_Tana.geojson\")"
```

Comment:
Hi Mahir, I currently do not have access to the geojson files. As such, I could not move ahead beyond the line in the review process.

Would it be possible to make some geojson files available within the github file for the example notebook? Overall, great work and happy coding :)